### PR TITLE
Installing Projection Wizard v2.1

### LINKED ISSUE #27: Projection Wizard 2.1 fixes
- [x] _Square format, equatorial azimuthal case:_ Limiting the case to square extents when an extent crosses the Equator (keeping the Equator a straight line only when it is displayed on a map). **(done in https://github.com/ProjectionWizard/projectionwizard.github.io/commit/d38100ebcf77e97ed476e4325c6c641693adf5c0)**
- [x] _Landscape format, cylindrical case:_  Fixing std. parallel value for all cases when a cylindrical projection is returned: **(done in https://github.com/ProjectionWizard/projectionwizard.github.io/commit/dccd84dd12e5999edf7d65bcf763d79fdcad8a77)**
      - When extent crosses or touches the Equator, a std. parallel in the middle between the Equator and the largest (absolute) latitude limit of an extent,
      - When extent does not crosses the Equator, a std. parallel is set in the middle of the extent (an alternative would be to return a conic projection instead),
      - No more cylindrical projection with std. parallel on the Equator.
- [x] Fixing _Lambert azimuthal equal area_'s name **(done in https://github.com/ProjectionWizard/projectionwizard.github.io/commit/3cc716ea628343dccf3fbf4eda200e8a1483ece9)**
- [x] Normalizing longitudinal projection parameters, like central meridian, to the range of [-180, 180]. **(done in https://github.com/ProjectionWizard/projectionwizard.github.io/commit/662066fe20beebfb32432e349843e967022fd788)**
- [x] Fixing web link to EPSG dataset **(done in https://github.com/ProjectionWizard/projectionwizard.github.io/commit/900fdcfed4d30bb4f5d9520c56c45523e5d250fc)**
- [x] Update Leaflet, D3, and jQuery dependencies **(done in https://github.com/ProjectionWizard/projectionwizard.github.io/commit/cc829eabcd7611d3ed844ab6644976694f1ce961 by @jwasilgeo)**

### PR DESCRIPTION
**Pull request by beuan**
This installs changes for Projection Wizard 2.1 listed in https://github.com/ProjectionWizard/projectionwizard.github.io/issues/27.

### CODE REVIEW

**Comment by beuan** [comment]
@danieljwood FYI

File: index.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <!--
-	PROJECTION WIZARD v2.0
+	PROJECTION WIZARD v2.1
 	Map Projection Selection Tool
 
-	Author: Bojan Savric
-	Date: June, 2020
+	Author: Bojan Savric, Jacob Wasilkowski
```
Comment (jwasilgeo) [inline index.html:5]: 🙇‍♂️ I'm honored!

File: map.js [hunk 2]
```diff
@@ -107,6 +107,17 @@ function outputLON(lon, ui_bool) {
 	return Math.round(lon * 1e7) / 1e7 + "º";
 }
 
+/* Normalizing longitude values */
+function normalizeLON(lon, lon0) {
```
Comment (jwasilgeo) [inline map.js:111]: 👍 nice to pull this out into a func and use throughout

**Review by jwasilgeo — APPROVED** [review]
Looks good! Very excited to see updates coming to 🗺️ 🌐 🧙.

**Review by ProjectionWizard — APPROVED** [review]
Looking great! Thank you, @jwasilgeo & @beuan!

**Pull request merged**